### PR TITLE
WIP Translations for keys of the specs

### PR DESCRIPTION
**Pull request by Ivans51**
Signed-off-by: Ivan Del Pino <idelpino@teclib.com>

### Changes description

Added values to the screen show inventory in the resource file

screenshot_1544724971

### Checklist

Please check if your PR fulfills the following specifications:

- [x] Tests for the changes have been added
- [ ] Docs have been added/updated

### Estimated time

|Assignee|:tomato:|
|:---|:---:|
|@ivans51|10|

### References

Closes: https://github.com/flyve-mdm/android-inventory-agent/issues/103
Related #N/A
Depends on #N/A

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -216,6 +216,27 @@
     <string name="InventoryFrequency">Frequency</string>
     <string name="InventoryFrequencyDialog">Every :</string>
     <string name="InventoryFrequencySummary">Define frequency</string>
+    <string name="Hardware">Hardware</string>
```
Comment (ajsb85) [inline app/src/main/res/values/strings.xml]: Add a separtor (comment)
Example: `<!--Inventory-->`

**Comment by Ivans51** [comment]
captura de pantalla 2018-12-14 a la s 3 10 48 p m

captura de pantalla 2018-12-14 a la s 3 10 52 p m

Different errors when rerun workflow process CI, is strange

**Comment by ajsb85** [comment]
Hi, @ivans51
Run the TX CLI tool in your local machine or re run the CircleCI with SSH access to debug the output of Transifex

**Comment by Ivans51** [comment]
Success in local machine, i will try by SSH

**Pull request merged**